### PR TITLE
Update azure-cosmos-db-v2.md

### PR DESCRIPTION
**Pull request by Rodrigossz**

### CODE REVIEW

**Review by revinjchalil — APPROVED** [review]

File: powerquery-docs/connectors/azure-cosmos-db-v2.md [hunk 1]
```diff
@@ -147,6 +147,12 @@ when processing a query. For example, in the following query, TOP is applied in
 
   * When the query has one or more partition keys appear in an IS NOT NULL clause in the WHERE clause.
 
+* The V2 connector doesn't support complex data types such as arrays, objects, and hierarchical structures. We recommend Fabric Mirroring for those scenarios.
```
Comment (revinjchalil) [inline powerquery-docs/connectors/azure-cosmos-db-v2.md:150]: Add the link to our Cosmos DB Mirroring documentation?

**Review by revinjchalil — APPROVED** [review]

File: powerquery-docs/connectors/azure-cosmos-db-v2.md [hunk 2]
```diff
@@ -147,6 +147,12 @@ when processing a query. For example, in the following query, TOP is applied in
 
   * When the query has one or more partition keys appear in an IS NOT NULL clause in the WHERE clause.
 
+* The V2 connector doesn't support complex data types such as arrays, objects, and hierarchical structures. We recommend the [Fabric Mirroring for Azure Cosmos DB](https://learn.microsoft.com/fabric/database/mirrored-database/azure-cosmos-db) feature for those scenarios.
```
Comment (seesharprun) [inline powerquery-docs/connectors/azure-cosmos-db-v2.md:150]:
```suggestion
* The V2 connector doesn't support complex data types such as arrays, objects, and hierarchical structures. We recommend the [Fabric Mirroring for Azure Cosmos DB](/fabric/database/mirrored-database/azure-cosmos-db) feature for those scenarios.
```

File: powerquery-docs/connectors/azure-cosmos-db-v2.md [hunk 3]
```diff
@@ -147,6 +147,12 @@ when processing a query. For example, in the following query, TOP is applied in
...
+  
+* The V2 connector uses sampling of the first 1,000 documents to come up with the inferred schema. It's not recommended for schema evolution scenarios when only part of the documents are updated. As an example, a newly added property to one document in a container with thousants of documents may not be inlcuded in the inferred schema. We recommend the [Fabric Mirroring for Azure Cosmos DB](https://learn.microsoft.com/fabric/database/mirrored-database/azure-cosmos-db) feature for those scenarios.
```
Comment (seesharprun) [inline powerquery-docs/connectors/azure-cosmos-db-v2.md:152]:
```suggestion
* The V2 connector uses sampling of the first 1,000 documents to come up with the inferred schema. It's not recommended for schema evolution scenarios when only part of the documents are updated. As an example, a newly added property to one document in a container with thousants of documents may not be inlcuded in the inferred schema. We recommend the [Fabric Mirroring for Azure Cosmos DB](/fabric/database/mirrored-database/azure-cosmos-db) feature for those scenarios.
```

**Comment by Rodrigossz** [comment]
#sign-off

**Review by revinjchalil — APPROVED** [review]

**Comment by Rodrigossz** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@revinjchalil 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Pull request merged**